test/javascript/bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "tests",
+  "version": "0.0.0",
+  "homepage": "",
+  "authors": [
+    ""
+  ],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "devDependencies": {
+    "jasmine": "~2.3.4"
+  }
```

Comment:
this makes it so you can do `bower install` in the test/javascript folder and it will download jasmine for you